package.json

```diff
@@ -29,8 +29,7 @@
   },
   "eslintConfig": {
     "extends": [
-      "react-app",
-      "react-app/jest"
+      "react-app"
```

Comment:
I think removing `react-app/jest` will cause VSCode to think there are errors in the test files. 

Is there a reason for removing this?

Reply:
I've put this back